ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -332,6 +332,68 @@ keys:
                 description: 'Action as string
 
                   Example: "permit 1b11:3a00:22b0:0082::/64 eq 128"'
+  mac_security:
+    type: dict
+    display_name: MACsec
+    keys:
+      license:
+        type: dict
+        required: true
+        keys:
+          license_name:
+            type: str
+            required: true
+          license_key:
+            type: int
```

Comment:
This should be str type not int. Also won't need the convert_types after this change.

Reply:
address in commit [57ecdf2](https://github.com/aristanetworks/ansible-avd/pull/2058/commits/57ecdf2f5950dad1e183c198de33926616fcf070)